### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/config/plugins/comment.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#624] [Announcements]
# https://theme-hope.vuejs.press/zh/config/plugins/comment.html

介绍 主题使用 vuepress-plugin-comment2 提供评论支持，并将主题选项中的 plugin.comment 作为插件选项传递。 vuepress-plugin-comment2 目前支持 4 种评论服务: Artalk; Giscus; Twikoo; Waline; 如果你不需要评论功能，请忽略此选项。 选项 对于每个评论服务对应...

https://theme-hope.vuejs.press/zh/config/plugins/comment.html

**Comment by OnlY0226** [comment]
test

**Comment by Friday-z2Z** [comment]
1

**Comment by dream-oyh** [comment]
请问博客已经按照文档里的流程配置好Waline，但是实际部署后提交评论会报错：failed to fetch，博客主题已经更新到最新版，Vercel绑定的GitHub仓库已经调整为public，我的博客地址是dream-oyh.pages.dev，请问能帮忙看看这个问题不，谢谢！

**Reply by Mister-Hope** [reply]
I need to authorize to visit any api at https://waline-3b4zisez7-dream-oyhs-projects.vercel.app/

This is obviously not expected, so check your own configuration to make the site public. If you can not do so, then it could be a vercel problem(changing) that breaks any services like waline to work on it.

**Reply by dream-oyh** [reply]
OK, let me see. Thanks!!
